File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.java
class Solution {
    private int count(int n){
        int cnt = 0;
        while(n != 1){
            n /= 2;
            cnt++;
        }
        return cnt;
    }
    public int solution(int[] num_list) {
        int ret = 0;
        for(int i : num_list) ret += count(i);
        return ret;
    }
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.py
def count(n):
    cnt = 0
    while n != 1:
        if n % 2 == 0: 
            n //= 2
        else: n = (n-1) //2
        cnt += 1
    return cnt

def solution(num_list):
    return sum(count(x) for x in num_list)
    